src/main/java/racingcar/validator/InputValidator.java

```diff
@@ -0,0 +1,95 @@
+package racingcar.validator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * 자동차 이름은 쉼표(,)를 기준으로 구분하며 이름은 5자 이하만 가능하다.
+ * 사용자가 잘못된 값을 입력할 경우 IllegalArgumentException을 발생시킨 후 애플리케이션은 종료되어야 한다.
+ */
+public class InputValidator {
+
+    private static final int MAX_CAR_NAME_LENGTH = 5;
+
+    private String delimitedCarNames;
+    private final List<String> carNames = new ArrayList<>();
+    private String attemptCount;
+
+    public List<String> validate(String delimitedCarNames, String attemptCount) {
+        this.delimitedCarNames = delimitedCarNames;
+        this.attemptCount = attemptCount;
+
+        validateCarNamesNull();
+        validateCarNamesFormat();
+        validateCarNameFormat();
+
+        validateIsNumber();
+        validateIsNegative();
+
+        return carNames;
+    }
+
+    private void validateIsNumber() {
+        try {
+            Integer.parseInt(attemptCount);
+        } catch (NumberFormatException e){
+            throw new IllegalArgumentException("시도 횟수로는 음이 아닌 정수를 입력해주세요");
+        }
+    }
+
+    private void validateIsNegative() {
+        if (Integer.parseInt(attemptCount) < 0) {
+            throw new IllegalArgumentException("시도 횟수로는 음이 아닌 정수를 입력해주세요");
+        }
+    }
+
+    private void validateCarNamesNull() {
+        if (delimitedCarNames == null) {
+            throw new IllegalArgumentException("자동차 이름 입력 형식이 null 입니다");
+        }
+    }
+
+    private void validateCarNamesFormat() {
+        int delimitedCarNamesLength = delimitedCarNames.length();
+        if (delimitedCarNames.startsWith(",")
+                || delimitedCarNames.isEmpty()
+                || isEndWithComma(delimitedCarNamesLength)) {
+            throw new IllegalArgumentException("올바른 자동차 이름 형식이 아닙니다");
+        }
+    }
+
+    private boolean isEndWithComma(int delimitedCarNamesLength) {
+        return delimitedCarNames.charAt(delimitedCarNamesLength - 1) == ',';
```

Comment:
구분자로 종료되는 것까지 꼼꼼한 검증 좋은데요?
```suggestion
        return delimitedCarNames.endsWith(",");
```
`endsWith()` 메서드로 나타내 볼 수 있을 것 같아요!